packages/web-components/examples/static-files/index.html

```diff
@@ -0,0 +1,127 @@
+<!--
+@license
+
+Copyright IBM Corp. 2020
+
+This source code is licensed under the Apache-2.0 license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+<html>
+  <head>
+    <title>@carbon/ibmdotcom-web-components example</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" type="text/css" href="//unpkg.com/carbon-components@10.20.0/css/carbon-components.css">
+    <style type="text/css">
+      @media (min-width: 66rem) {
+        .bx--offset-lg-3 {
+          margin-left: 0;
+        }
+      }
+
+      .bx--content.dds-ce-demo--ui-shell-content {
+        margin-top: 3rem;
+      }
+
+      .bx--content.dds-ce-demo--ui-shell-content h2 {
+        margin: 30px 0;
+      }
+
+      .bx--content.dds-ce-demo--ui-shell-content h2:first-of-type {
+        margin-top: 0;
+      }
+    </style>
+    <script type="module">
+      // Copy from `dist` and put it to the same directory as this file
+      import './ibmdotcom-web-components-dotcom-shell.min.js';
+    </script>
+  </head>
+  <body>
+    <dds-masthead>
```

Comment:
@asudoh does the bundle also include `dds-dotcom-shell-container`? That should probably be the `quick start` version of this, followed by the customized `masthead` for more advanced implementations.

Reply:
@jeffchew Yes. I started with manual approach due to `CORS_PROXY` thing, van update this once our fix to its reliance us in.

Reply:
You can probably already get started on switching over, the access control policy is already live on EI server.

Reply:
🎉 - Updated the examples.